build.gradle

```diff
@@ -16,10 +16,13 @@
 allprojects {
   apply plugin: 'java-library'
 
-  sourceCompatibility = 11
-  targetCompatibility = 11
+  sourceCompatibility = 17
```

Comment:
21?

Reply:
re: removing matter-labs, I was thinking we can remove it once we have a besu release with gnark, I have a PR ready to put up for that, it is just waiting for a release of besu-native that with this PR

Reply:
thanks, made it 21 and bumped the project version to next minor rev to signify